src/index.js

```diff
@@ -1236,15 +1234,15 @@ const announcement = new Announcement();
 
             return;
         });
-        
+
         // POST event from /envvar form
         app.post('/envvar', async ({ body }, res) => {
             const { password } = /** @type {{ password:string }} */(body);
 
             const validPwd = password === process.env.PASSWORD;
 
             // Convert request to JSON object - see https://stackoverflow.com/a/8649003
-            const envVars = JSON.parse('{"' + body.replace(/&/g, '","').replace(/=/g,'":"') + '"}', function(key, value) { return key===""?value:decodeURIComponent(value) });
+            const envVars = JSON.parse('{"' + body.replace(/&/g, '","').replace(/=/g, '":"') + '"}', function (key, value) { return key === "" ? value : decodeURIComponent(value); });
```

Comment:
This has been removed in my latest changes, probably what's causing the conflict.

Reply:
Don't worry, I'll rebase :) Constantly do, so should not be a problem